src/components/organisms/Chatbox/Chatbox.tsx

```diff
@@ -65,11 +77,35 @@ const Chatbox: React.FunctionComponent<PropsType> = ({
       chatsToDisplay &&
       chatsToDisplay.filter(
         (chat: any) =>
-          (chat.from === discussionPartner.id && chat.to === user.uid) ||
-          (chat.to === discussionPartner.id && chat.from === user.uid)
+          (chat.from === discussionPartner?.id && chat.to === user.uid) ||
+          (chat.to === discussionPartner?.id && chat.from === user.uid)
       );
   }
 
+  let privateRecipient;
+  const lastChat = chatsToDisplay && chatsToDisplay[0];
+  if (
+    !isPrivate &&
+    !hasUserSelectedRecipient &&
+    lastChat &&
+    lastChat.type === "private"
+  ) {
+    if (lastChat?.to === user.uid) {
```

Comment:
I don't know how active the chat is, but it shouldn't change who you're sending it to while you're writing something.

Reply:
Seems like a pretty reasonable point, lets raise this point to chat with Ed and I in TR

Reply:
Ah just noticed this got addressed as I kept reading, please disregard! Great work!